### PR TITLE
Update people.yml: bio, education, avatar

### PR DESCRIPTION
**Pull request by 8bitsam**

### CODE REVIEW

File: db/people.yml [hunk 1]
```diff
@@ -3969,9 +3969,13 @@ rzyu:
...
-  bio: bio
-  education: []
+  avatar: "https://media.licdn.com/dms/image/D4E03AQFXa02w443ZCg/profile-displayphoto-shrink_200_200/0/1667414532631?e=1695254400&v=beta&t=o2XvkKEklVgkrx646E2f8PGkxGHjU2r7CfTjkJ1He0Q"
+  bio: Rising Junior at Columbia Universit studying Applied Physics with a minor in CS. Interests in all types of computational physics.
+  education:
+    - institution: Columbia University (SEAS)
+    - degree: BS in Applied Physics with a minor in Computer Science
+    - begin_year: 2022
+    - end_year: 2025
   email: "\x16sqa2001@columbia.edu"
```
Comment (sbillinge) [inline db/people.yml:3979]: please remove the \x16 which is a rogue utf8 code that crept in.

File: db/people.yml [hunk 2]
```diff
@@ -3969,9 +3969,13 @@ rzyu:
 sandrello:
   active: true
   aka: []
-  avatar: "https://github.com/\x168bitsam.png"
-  bio: bio
-  education: []
+  avatar: "https://media.licdn.com/dms/image/D4E03AQFXa02w443ZCg/profile-displayphoto-shrink_200_200/0/1667414532631?e=1695254400&v=beta&t=o2XvkKEklVgkrx646E2f8PGkxGHjU2r7CfTjkJ1He0Q"
+  bio: Rising Junior at Columbia Universit studying Applied Physics with a minor in CS. Interests in all types of computational physics.
```
Comment (sbillinge) [inline db/people.yml:3973]: typo?

**Review by sbillinge — COMMENTED** [review]
looks great, thanks!  please see a couple of comments inline.

just make those edits and push and they will appear on the  same PR

**Comment by sbillinge** [comment]
@8bitsam please make sure that your commit messages are meaningful. It should say what changes were made in that commit.

**Review by sbillinge — COMMENTED** [review]
@8bitsam your pr is failing CI due to errors.  Please click on the "Details" link next to the red x or dot in the conversation tab to see the issue.

It shows up as a validation error in the db, but actually it is a formatting error.  Please pay close attention and see if you can figure out what is wrong.  Maybe spend a few minutes reading about yaml and understanding what all the indents and symbols mean in terms of the structure of the objects in the fil.

**Review by sbillinge — COMMENTED** [review]
I think this will be compliant, but please also check the standards we are using for indentation.  I think each level we indent by 2, so the list dash should go in by two spaces, but check the other entries in the file and conform. 

This is called linting....it doesn't take much time or overhead once you get into the habit and it makes code much more readable and lowers cognitive overload later, so we take it seriously, especially with the python code itself (but also hte yaml)

**Comment by 8bitsam** [comment]
I think it should be good now, following the standards and all

**Comment by sbillinge** [comment]
I will merge when it passes CI

**Comment by sbillinge** [comment]
nice work!

**Pull request merged**